OpenSim/Examples/DataTable/example2.cpp

```diff
@@ -0,0 +1,82 @@
+/* -------------------------------------------------------------------------- *
+ *                            OpenSim:  example2.cpp                          *
```

Comment:
I like different files because they can stand independently and users can just copy the whole file if they want to play with it. This is how other projects usually do it. Putting all the examples in one file can also create problems for naming the variables in the same scope. Different examples then will have to be split up into functions and so on.
I can change to have just one file but I prefer different files under one directory.

The two examples follow the same steps but the first one creates a TimeSeriesTable at the beginning and second one creates a DataTable at the beginning.